tests/settings/base.py

```diff
@@ -9,6 +9,8 @@
     'localhost',
 ]
 
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Comment:
Looks like this is actually the default value, so I don't think we need to add it. [docs](https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field)

Reply:
For this one, it gives a warning in the terminal. I thought I should set it so that it's not missed once they do change it. The 3.2 release notes say, `In a future Django release the default value of DEFAULT_AUTO_FIELD will be changed to BigAutoField.`. I can remove it if it's something that should be changed when the time comes.

Reply:
ah, I hadn't seen that note, but I did see something in the docs about there possibly being a problem if the default changes later, so nvm, I think you've made the best decision to include it